Remotion/Core/Reflection.CodeGeneration.UnitTests/Core.Reflection.CodeGeneration.UnitTests.csproj

```diff
@@ -114,14 +114,15 @@
       <HintPath>..\..\..\packages\CommonServiceLocator.1.2\lib\portable-windows8+net40+sl5+windowsphone8\Microsoft.Practices.ServiceLocation.dll</HintPath>
       <Private>True</Private>
     </Reference>
+    <Reference Include="Moq, Version=4.16.0.0, Culture=neutral, PublicKeyToken=69f491c39445e920">
+      <HintPath>..\..\..\packages\Moq.4.16.1\lib\net45\Moq.dll</HintPath>
+      <Private>True</Private>
+    </Reference>
+    <Reference Include="mscorlib" />
```

Comment:
Why directly reference mscorlib?

Reply:
this also gets inserted automatically, but i will delete it since it's not needed

Reply:
ah, okay, in that case...

Reply:
ahhh ok, this reference was not inserted automatically by Castle/Moq, I inserted it by mistake while fixing the unit tests after the update